repos/iiif-storage/src/Aggregate/DereferencedManifest.php

```diff
@@ -70,11 +70,31 @@ public function mutate(ItemRequest $input)
                 }
 
                 $title = $input->getValue('dcterms:title');
-                if (!$title || empty($title) || current($title)->getValue() === '') {
-                    $label = $this->translate($manifest['label'] ?? '');
+                if (!$title || empty($title) || trim(current($title)->getValue()) === '') {
+                    $label = $manifest['label'] ?? null;
                     if ($label) {
-                        $input->addField(
-                            FieldValue::literal('dcterms:title', 'Title', $label)
+                        $input->addFields(
+                            FieldValue::literalsFromRdf('dcterms:title', 'Title', $label)
+                        );
+                    }
+                }
+
+                $description = $input->getValue('dcterms:description');
+                if (!$description || empty($description) || trim(current($description)->getValue()) === '') {
+                    $summary = $manifest['description'] ?? null;
+                    if ($summary) {
+                        $input->addFields(
+                            FieldValue::literalsFromRdf('dcterms:description', 'Summary', $summary)
+                        );
+                    }
+                }
+
+                $attribution = $input->getValue('sc:attributionLabel');
+                if (!$attribution || empty($attribution) || trim(current($attribution)->getValue()) === '') {
+                    $attribution = $manifest['attribution'] ?? null;
```

Comment:
Need some abstraction here like:
```
$input->replaceIfEmpty( ... );
```
to avoid this verbose block being used all over the place